setup.py

```diff
@@ -569,6 +569,7 @@ def run(self):
         setuptools.command.build_ext.build_ext.run(self)
 
         if IS_DARWIN and package_type != 'conda':
+            report('-- Building libiomp')
```

Comment:
I think this is misleading, as it's not building libiomp, but bundles it into the package

Reply:
amended